pkg/distributor/ratestore.go

```diff
@@ -62,10 +56,10 @@ type rateStore struct {
 	maxParallelism         int
 	rateRefreshFailures    *prometheus.CounterVec
 	refreshDuration        *instrument.HistogramCollector
-	overrides              overrides
+	overrides              Limits
```

Comment:
nit: maybe other Loki maintainers will be confused with overrides being used as limits here?